hawkpost/static/javascripts/global.js

```diff
@@ -16,11 +16,11 @@ $(document).ready(function(){
     popup.classList.toggle('show');
   })
 
-$(".faq__box--big").click(function(){
-    $(this).toggleClass("faq__box");
+$(".faq__box").click(function(){
+    //$(this).toggleClass("faq__box");
```

Comment:
Instead of commenting, if this is no longer necessary please remove it. Same for line 23.